stdlib/public/runtime/Private.h

```diff
@@ -183,6 +183,8 @@ namespace swift {
                              void *data);
   uint8_t *_swift_getSectionDataPE(void *handle, const char *sectionName,
                                    unsigned long *sectionSize);
+#endif
+#if defined(__CYGWIN__)
   void _swift_once_f(uintptr_t *predicate, void *context,
```

Comment:
So the ABI of `std::once_flag` on MSVC happens to match what we expect?  If so, please extend the comment in `stdlib/public/runtime/Once.cpp` to mention that MSVC port also relies on this, not just libstdc++-based ports.